File: src/main/scala/arimitsu/sf/cql/v3/util/BufferUtil.scala
package arimitsu.sf.cql.v3.util

import java.nio.ByteBuffer

/**
 * Created by sxend on 14/06/08.
 */
object BufferUtil {
  def length(buffer: ByteBuffer): Int = {
    buffer.limit() - buffer.position()
  }
}

File: src/main/scala/arimitsu/sf/cql/v3/util/BufferUtil.java
package arimitsu.sf.cql.v3.util;

import java.nio.ByteBuffer;

/**
 * Created by sxend on 14/06/08.
 */
public class BufferUtil {
    public static int length(ByteBuffer buffer) {
        return buffer.limit() - buffer.position();
    }
}
